app/controllers/users/personal_keys_controller.rb

```diff
@@ -51,7 +51,7 @@ def user_has_not_visited_any_sp_yet?
     # @return [FormResponse]
     def send_new_personal_key_notifications
       emails = current_user.confirmed_email_addresses.map do |email_address|
-        UserMailer.personal_key_regenerated(email_address.email).deliver_now
+        UserMailer.personal_key_regenerated(current_user, email_address.email).deliver_now
```

Comment:
This isn't super important, but some of the emails are `deliver_now` and some are `deliver_later`, is there a reason for that?

Reply:
I have no idea 🤷